src/scicookie/hooks/post_gen_project.py

```diff
@@ -259,9 +258,9 @@ def http2ssh(url):
 
 
 def validation():
-    if USE_BLUE and USE_BLACK:
+    if USE_BLACK:
         raise Exception(
-            "The libs Blue and Black were selected, but you need to choose "
+            "The libs Black were selected, but you need to choose "
             "just one of them."
         )
```

Comment:
This whole `if` is now not required. It was written to make sure that the user does not choose black and blue together.

Reply:
Implemented